frontend/src/app/app.component.ts

```
@@ -0,0 +1,26 @@
+export class AppComponent implements OnInit {
+  constructor(
+    private readonly themeService: ThemeService
+  ) {
+
+  }
+
+  ngOnInit(): void {
+    const localStorageTheme = this.themeService.getLocalStorageTheme();
+    if (localStorageTheme) {
+      this.themeService.switchTheme(localStorageTheme);
+    } else if (window.matchMedia && window.matchMedia('(prefers-color-scheme: dark)').matches) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)